src/components/YesNoChoice.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { RadioButtonGroup, RadioButton } from 'material-ui';
+
+const groupStyle = {
+  display: 'flex',
+  borderLeft: '1px solid #d1d1d1'
+};
+
+const buttonStyle = {
+  width: 'auto',
+  borderWidth: '1px 1px 1px 0',
+  borderStyle: 'solid',
+  borderColor: '#d1d1d1',
+  padding: "10px 20px"
+};
+
+/*
+  A component implementing a yes/no choice.
+  */
+const YesNoChoice = (props) => (
+  <RadioButtonGroup
+    name={props.name}
+    valueSelected={props.value}
```

Comment:
Do I interpret this correctly in that the value of a YesNoChoice is true or false? Perhaps rename to BooleanChoice? (Even if the user sees yes/no, the programmer sees a boolean value.)